test/support/document_controller_test_helpers.rb

```diff
@@ -308,93 +308,7 @@ def should_show_inapplicable_nations(document_type)
       end
     end
 
-    def should_paginate(edition_type, options = {})
```

Comment:
Should all of this removal be in this commit or is the pagination one more suitable?

Reply:
I think this should stay here, as it's a helper for view tests where the view has been deleted.  There's no further use of this helper elsewhere, so I deleted it at the same time as the view tests.